.github/workflows/test.yaml

```diff
@@ -0,0 +1,68 @@
+name: test
+on:
+  push:
+    branches:
+      - main
+jobs:
+  test_dashboard_server:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run tests
+        run: ./test.sh dashboard-server --is-ci
+  test_dean:
```

Comment:
The tests for individual apps should only be run on changes to their folder, no? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore